doc/capitulos/metodologia/introduccion.tex

```diff
@@ -0,0 +1,150 @@
+% !TeX root = ../../tfg.tex
+% !TeX encoding = utf8
+%
+%*******************************************************
+% Metodología
+%*******************************************************
+
+\chapter{Metodología}
+
+La planificación y organización es un componente vital a la hora del desarrollo de software 
+y la ciencia de datos. Por lo que no es de extrañar que sus beneficios sean extrapolables
+a otras áreas de la ciencia; 
+hemos aplicado por ende la 
+metodología expuesta en el artículo \cite{DBLP:journals/corr/abs-2104-12545}, realizado 
+el trabajo siguiendo una metodología ágil, de acorde a las siguientes hipótesis. 
+
+\begin{enumerate}
+    \item Se premia la \textbf{reproducibilidad}, \textit{la ciencia no puede ser ciencia si no es reproducible}. Es por 
+    ello que se publica todo el material usado. 
+    \item \textbf{Comprobaciones} en todo los niveles, \textit{el código que no se ha probado no funciona}. 
+    Explicitamos unos requisitos y criterios claros de validación. 
+    \item Conocimiento libre, tanto la memoria como el código se encuentra publicado y con licencia libre en nuestro repositorio 
+    de github \cite{TFG-Estudio-de-las-redes-neuronales}.
+
+    \textcolor{red}{ JJ, el cuarto punto de tu \cite{DBLP:journals/corr/abs-2104-12545} artículo: "Stakeholder collaboration over vertical chains-of-command" 
+    no lo he puesto porque no creo que encaje con la situación, el rol profesor-alumno necesariamente tiene cierto 
+    enfoque vertical, no por la autoridad; sino por la desigualdad de conocimientos, conduciendo a esa relación "desigual".
+     }
+\end{enumerate}  
+
+\section{Descripción de la metodología }  
+
+Concretamente la metodología seguida basada en \cite{que-es-un-trabajo-fin-de-x} ha sido la siguiente: 
+
+El objetivo principal es resolver un problema usando técnicas tanto informáticas como matemáticas. Para ello 
+se han definido unos objetivos claros a través de una \textit{metodología de personas}  \cite{personas-why-and-how-you-should-use-them}
+y las respectivas historias de usuario derivadas.   
+
+\subsection{Personas definidas}  
+
+Las personas que hemos definido son:  
+
+\begin{itemize}
+    \item \textbf{Perfil de usuario Tribunal}
```

Comment:
Una "persona" debe tener nombre y apellidos, mira aquí https://www.interaction-design.org/literature/article/personas-why-and-how-you-should-use-them
En este caso, el que sea de informática típicamente, por ejemplo, es la justificación para que en la memoria vayan explicaciones informáticas de los resultados matemáticos.